Content.Client/UserInterface/GhostRoleWindow.cs

```diff
@@ -0,0 +1,16 @@
+using Content.Client.GameObjects.EntitySystems;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Shared.GameObjects.Systems;
+
+namespace Content.Client.UserInterface
+{
+    public class GhostRoleWindow : SS14Window
+    {
+        protected override void Opened()
+        {
+            base.Opened();
+
+            EntitySystem.Get<GhostRoleSystem>()
```

Comment:
very epic

Reply:
reee this is a draft I still gotta work on the UI itself

Reply:
naughty naughty